bitmap-pegastickacorn/index.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const bitmapFileHelper = require('./lib/bitmap-file-helper');
+
+
+bitmapFileHelper.readFile('bitmap', function(err, bitmap) {
+  bitmap.invertColor();
+  bitmapFileHelper.writeFile('invert', 'bitmap', bitmap, function(err, data){
+  });
+});
+
+bitmapFileHelper.readFile('bitmap', function(err, bitmap) {
+  bitmap.greyScaleColor();
+  bitmapFileHelper.writeFile('greyscale', 'bitmap', bitmap, function(err, data){
+  });
+});
```

Comment:
You're having linter errors because you're not using err or data parameters. You could try this:
```
bitmapFileHelper.readFile('bitmap', function(err, bitmap) {
  bitmap.greyScaleColor();
  bitmapFileHelper.writeFile('greyscale', 'bitmap', bitmap, function(err){
    console.error(err);
  });
});
```